elements/pl-file-editor/pl-file-editor.py

```diff
@@ -6,6 +6,15 @@
 import os
 
 
+EDITOR_CONFIG_FUNCTION_DEFAULT = None
+ACE_MODE_DEFAULT = None
+ACE_THEME_DEFAULT = None
+SOURCE_FILE_NAME_DEFAULT = None
+MIN_LINES_DEFAULT = None
+MAX_LINES_DEFAULT = None
+AUTO_RESIZE_DEFAULT = 'true'
```

Comment:
This enables the `auto-resize` behavior by default, which was the intended effect of #1656 (c.f. documentation).

Reply:
Sounds good to me.